packages/flutter_tools/lib/src/android/gradle_errors.dart

```diff
@@ -634,15 +635,15 @@ final GradleHandledError couldNotOpenCacheDirectoryHandler = GradleHandledError(
   eventLabel: 'could-not-open-cache-directory',
 );
 
-
 String _getAgpLocation(FlutterProject project) {
   return '''
  The version of AGP that your project uses is likely defined in:
 ${project.android.settingsGradleFile.path},
-in the 'plugins' closure.
+in the 'plugins' closure (by the number following "com.android.application").
  Alternatively, if your project was created with an older version of the templates, it is likely
 in the buildscript.dependencies closure of the top-level build.gradle:
-${project.android.hostAppGradleFile.path}.''';
+${project.android.hostAppGradleFile.path},
+by the number following "com.android.tools.build:gradle:".''';
```

Comment:
Nit: "as"->"by" (since this sentence doesn't have "defined" earlier, like the other one, and "by" doesn't work without it.)

Reply:
changed back